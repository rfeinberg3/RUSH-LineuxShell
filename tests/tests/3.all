rush> Hello World
rush> rush> An error has occurred
rush> Hello World
rush> rush> An error has occurred
rush> 