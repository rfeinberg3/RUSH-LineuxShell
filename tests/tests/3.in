echo Hello World
echo Hello World > /tmp/tmp.txt
echo Hello World > /tmp/tmp.txt > /tmp/tmp.txt
echo Hello World
      
exit with arguments
exit
