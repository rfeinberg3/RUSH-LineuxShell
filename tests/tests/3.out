rush> Hello World
rush> rush> rush> Hello World
rush> rush> rush> 